ddtrace/appsec/ddwaf/__init__.py

```diff
@@ -0,0 +1,95 @@
+import ctypes
+import time
+from typing import Any
+from typing import Union
+
+from ddtrace.internal.compat import PY3
+
+from .ddwaf_types import ddwaf_config
+from .ddwaf_types import ddwaf_context_destroy
+from .ddwaf_types import ddwaf_context_init
+from .ddwaf_types import ddwaf_destroy
+from .ddwaf_types import ddwaf_get_version
+from .ddwaf_types import ddwaf_init
+from .ddwaf_types import ddwaf_object
+from .ddwaf_types import ddwaf_result_free
+from .ddwaf_types import ddwaf_ruleset_info
+from .ddwaf_types import py_ddwaf_required_addresses
+from .ddwaf_types import py_ddwaf_run
+
+
+# Python 2/3 unicode str compatibility
+if PY3:
+    unicode = str
```

Comment:
Maybe use https://six.readthedocs.io/#six.ensure_text ?

Reply:
I'm new here so maybe I don't know the usual practices but I wonder: is it really worth to add a dependency to an external library for something we can properly do in two lines of python ?

Reply:
We already have `six`.